tests/jballerina-unit-test/src/test/resources/test-src/types/anydata/anydata_test.bal

```diff
@@ -880,6 +882,8 @@ function testRuntimeIsAnydata() {
     assertFalse(g is anydata);
     assertFalse(h is anydata);
     assertFalse(i is anydata);
+    assertFalse(j is anydata);
+    assertFalse(k is anydata);
```

Comment:
```
assertFalse(table [{"e": error("Message")}] is anydata);
assertFalse(table [{"a": {"x": error("Message")}}] is anydata);
```
Above is-expr checks provide compilation errors with the fix. But ideally it shouldn't right?